demo/App.js

```diff
@@ -2,11 +2,13 @@ import React from "react";
 import { AppLoading, Asset, Font } from "expo";
 import PropTypes from "prop-types";
 import AppNavigator from "./AppNavigator";
-import { Action } from "tasit-sdk";
-import tasitSdkConfig from "./config/default.js";
 import { Provider } from "react-redux";
 import { createStore } from "redux";
 import demoApp from "./redux/reducers";
+import { Action } from "tasit-sdk";
+const { ConfigLoader } = Action;
+import tasitSdkConfig from "./config/default.js";
+import { checkBlockchain } from "./helpers.js";
```

Comment:
We can do without the .js suffix if we like

Reply:
Yes, sure.

Reply:
Done.

Reply:
Thanks. Resolving.